resources/data/dictionary/inputs.csv

```diff
@@ -132,6 +132,7 @@ build_existing_model.orientation,:ref:`orientation`
 build_existing_model.overhangs,:ref:`overhangs`
 build_existing_model.plug_load_diversity,:ref:`plug_load_diversity`
 build_existing_model.plug_loads,:ref:`plug_loads`
+build_existing_model.project_name,The name of the housing characteristics project.

```

Comment:
Put this a bit lower in the list so it's organized along with the other `build_existing_model.xxx` entries which are not actually tsv parameters?